javascript/MaterialXView/source/index.js

```diff
@@ -179,8 +186,19 @@ function animate()
         viewer.getScene().setUpdateTransforms();
     }
 
-    renderer.render(viewer.getScene().getScene(), viewer.getScene().getCamera());
-    viewer.getScene().updateTransforms();
+    // Only re-render when an update request was made
```

Comment:
Re-render gated by requirement to refresh. All the `schedulUpdate()` calls triggers the requirment.